File: Algorithms/C#/CH0/Hanoi.cs
using System;

namespace Hanoi
{
    class Program
    {
        static void Hanoi(int A, int B, int C, int n){
            if(n<=0) return;
            Hanoi(A,C,B,n-1);
            Console.WriteLine("Move Disk#{0} from Peg[{1}] to Peg[{2}]",n,A,C);
            Hanoi(B,A,C,n-1);
        }
        static void Main(string[] args)
        {
            Hanoi(1,2,3,3);
        }
    }
}

File: Algorithms/JAVA/CH0/Hanoi.java
public class Hanoi{
    public static void hanoi(int A,int B,int C,int n){
        if(n==0) return;
        hanoi(A,C,B,n-1);
        System.out.printf("Move Disk #%d from Peg[%d] to Peg[%d]\n",n,A,C);
        hanoi(B,A,C,n-1);
    }
    public static void main(String args[]){
        hanoi(1,2,3,3);
    }
}